pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -106,6 +108,7 @@ type Config struct {
 		ApiKey     string `gcfg:"api-key"`
 		TenantId   string `gcfg:"tenant-id"`
 		TenantName string `gcfg:"tenant-name"`
+		TrustId    string `gcfg:"trust-id"`
```

Comment:
don't we also need to populate AuthOptions.TokenID if you want to use passwordless TrustId? Just looking through AuthenticateV3Trust/trustv3auth in the godeps